common/src/main/java/com/wynntils/features/user/overlays/ShamanTotemTrackingFeature.java

```diff
@@ -46,13 +46,13 @@ public class ShamanTotemTrackingFeature extends UserFeature {
     public boolean highlightShamanTotems = true;
 
     @Config
-    public static ChatFormatting firstTotemColor = ChatFormatting.WHITE;
+    public static TotemTrackingColors firstTotemColor = TotemTrackingColors.WHITE;
```

Comment:
Why not CustomColor and pass it to font renderer?

Reply:
We can't set a team color to CustomColor, it has to be a ChatFormatting code

Reply:
Oh, I wanna try injecting into the renderer code then to make it able to render any color. I will push that to this PR when I get it to work, alright? Or do you wanna try to so that?

Reply:
Feel free to push to here, I dunno if I can get that working anytime soon as the only mixins I've touched are for sending events